### ISSUE TITLE
Warning within admin dashboard

### ISSUE THREAD

**Issue by vinkla** [iseqqavoq/APSIS-Pro-for-WP#2]
When debug is enabled we're seeing this error with WordPress 4.4.2:

```
( ! ) Warning: call_user_func() expects parameter 1 to be a valid callback, class 'APSIS_Pro_For_WP' does not have a method 'apsispro_hidden_settings_section_callback' in .../wordpress/wp-admin/includes/template.php on line 1403
```

We've only installed and activated the plugin.

**Comment by mattiasiqq** [comment]
Issue solved in 1.0.3

**Comment by vinkla** [comment]
:+1:
